src/main/java/com/questhelper/helpers/quests/thepathofglouphrie/ThePathOfGlouphrie.java

```diff
@@ -0,0 +1,654 @@
+		informKingBolren.addTeleport(teleToBolren);
+
+		var teleToStronghold = new TeleportItemRequirement("Spirit tree to Gnome Stronghold [2]", -1, -1);
+
+		// Talk to Gianne Junior in Tree Gnome Stronghold
+		talkToGianneJnr = new NpcStep(this, NpcID.GIANNE_JNR, new WorldPoint(2439, 3502, 1), "Talk to Gianne jnr. in Tree Gnome Stronghold to ask for Longramble's whereabouts.");
+		climbUpToGianneJnr = new ObjectStep(this, ObjectID.LADDER_16683, new WorldPoint(2466, 3495, 0), "");
+		climbUpToGianneJnr.setText(talkToGianneJnr.getText());
+		climbUpToGianneJnr.addTeleport(teleToStronghold);
+		talkToGianneJnr.addSubSteps(climbUpToGianneJnr);
+		talkToGianneJnr.addDialogSteps("I need your help finding a certain gnome.");
+
+		/// Find Longramble
+		var teleToLongramble = new TeleportItemRequirement("Fairy Ring BKP or Ring of Dueling to Castle Wars", ItemCollections.RING_OF_DUELINGS, 1);
+		teleToLongramble.addAlternates(ItemCollections.FAIRY_STAFF);
+
+
+		var goToLongramble = new ObjectStep(this, ObjectID.TREE_49590, new WorldPoint(2333, 3081, 0), "");
+		goToLongramble.addRecommended(earmuffsOrSlayerHelmet);
+		goToLongramble.addDialogStep("Castle Wars Arena.");
+		goToLongramble.addTeleport(teleToLongramble);
+		var actuallyTalkToLongramble = new NpcStep(this, NpcID.LONGRAMBLE, new WorldPoint(2340, 3094, 0), "");
+		actuallyTalkToLongramble.addRecommended(earmuffsOrSlayerHelmet);
+
+
+		talkToLongramble = new ConditionalStep(this, goToLongramble, "Go to Longramble, make sure to head to a bank & gear up first. You can drop all leftover discs.", combatGear, crossbow.equipped().highlighted(), mithGrapple.equipped().highlighted(), prayerPotions, food, crystalChime);
+		talkToLongramble.addStep(nearLongramble, actuallyTalkToLongramble);
+
+		{
+			var talk = new ObjectStep(this, NullObjectID.NULL_49598, new WorldPoint(2339, 3111, 0), "");
+			talkToSpiritTree = new ConditionalStep(this, talk, "Talk to the Spirit Tree", combatGear, prayerPotions, food, crystalChime);
+		}
+
+		useCrystalChime = new ObjectStep(this, NullObjectID.NULL_49598, new WorldPoint(2339, 3111, 0), "Use the Crystal Chime on the Spirit Tree", crystalChime.highlighted());
+		useCrystalChime.addIcon(ItemID.CRYSTAL_CHIME);
+
+		{
+			var talk = new ObjectStep(this, NullObjectID.NULL_49598, new WorldPoint(2339, 3111, 0), "");
+			talkToSpiritTreeAgain = new ConditionalStep(this, talk, "Talk to the Spirit Tree again", combatGear, prayerPotions, food, crystalChime);
+		}
+
+		/// The Warped Depths
+		enterSewer = new ObjectStep(this, ObjectID.SEWER_ENTRANCE, new WorldPoint(2322, 3101, 0),
+			"Enter the sewer to the west of the Spirit tree");
+		enterSewer.addRequirement(combatGear, prayerPotions, food, crystalChime);
+
+		sewer1Ladder = new ObjectStep(this, ObjectID.LADDER_49700, "Climb up the ladder");
+		sewer2Ladder = new ObjectStep(this, ObjectID.LADDER_49701, new WorldPoint(1529, 4236, 1),
+			"Climb down the ladder.");
+		sewer2Ladder.addRecommended(protectMissiles);
```

Comment:
If the earmuffs reduce damage worth us highlighting to wear them for this section too as recommended.

Reply:
Fixed in [`24f1930` (#1289)](https://github.com/Zoinkwiz/quest-helper/pull/1289/commits/24f1930ba8547df9b20c1ec6e22185bb387e5b64)